frontend/testing/requests/lesson.test.js

```
@@ -0,0 +1,579 @@
+  });
+
+  describe("getLessonBySlug", () => {
+    it("successfully fetches a lesson by slug", async () => {
+      const mockLesson = {
+        id: 1,
+        name: "Test Lesson",
+        slug: "test-lesson",
+        blocks: [],
+      };
+
+      const { fetchAPI } = require("@/lib/utils");
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)